src/api/settings/getServerSettings.js

```diff
@@ -34,6 +34,7 @@ export type ApiResponseServerSettings = {|
   realm_name: string,
   realm_uri: string,
   require_email_format_usernames: boolean,
+  zulip_feature_level: number,
```

Comment:
I think in this case if the server's response has no `zulip_feature_level`, I'd like to translate that to a value of 0 in the `Account` object. The reason is that if the server doesn't send a `zulip_feature_level`, that means it predates the introduction of that property -- which can be treated implicitly as one very broad feature level of 0, preceding the first explicit feature level which is 1.

We'll still need the type to be `number | null`, I think, to handle accounts that the user hasn't looked at since upgrading to an app version that stores feature levels.

Reply:
See docs at https://zulipchat.com/api/server-settings :
> New in Zulip 2.2. We recommend using an implied value of 0 for Zulip servers that do not send this field.

Reply:
Hmm, that does make sense! So, I think that would mean the `REALM_INIT` and `REALM_ADD` actions have `zulipFeatureLevel: number`, while the `Account` still has `number | null`, right?

Reply:
Yep, that sounds right!

Reply:
Err, actually `REALM_ADD` will, as it's based on `server_settings` -- but `REALM_INIT` only needs the initial data from `/register` (its `realm` property), per my other comment just now.